modules/navier_stokes/examples/solidification/galium_melting.i

```diff
@@ -180,11 +180,16 @@ Ny = 50
     pressure = pressure
   []
   [u_friction]
-    type = INSFVMomentumFriction
+    type = PINSFVMomentumFriction
     variable = vel_x
     momentum_component = 'x'
-    linear_coef_name = 'Darcy_coefficient'
-    quadratic_coef_name = 'Forchheimer_coefficient'
+    u = vel_x
+    v = vel_y
+    Darcy_name = 'Darcy_coeff'
+    Forchheimer_name = 'Forchheimer_coeff'
+    rho = ${rho_liquid}
+    mu = ${mu}
+    use_standard = false
```

Comment:
Use standard formulation would be more descriptive.

Reply:
renamed the flag to make it more clear.